hack/gen-clusterclass-test-flavors.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+#set -o verbose
+
+root=$(dirname "${BASH_SOURCE[0]}")/..
+kustomize="${root}/hack/tools/bin/kustomize"
+test_dir_path="test/e2e/data/infrastructure-aws"
+test_dir="${root}/${test_dir_path}/withclusterclass"
```

Comment:
As the `withclusterclass` appears to be the only difference to **gen-test-flavours** could we use the same script but with an argument? Instead of duplicating the script file?

Reply:
Fixed.